Frameworks/UIKit/UIButton.mm

```diff
@@ -182,7 +182,7 @@ - (void)_UIButton_initInternal:(WXFrameworkElement*)xamlElement {
     // Set the XAML element's name so it's easily found in the VS live tree viewer
     [_xamlButton setName:[NSString stringWithUTF8String:object_getClassName(self)]];
 
-    __weak UIButton* weakSelf = self;
+    __block UIButton* weakSelf = self;
```

Comment:
UIButton is not ARC-enabled. File a bug so we can move it to ARC-enabled and then the "weak" decoration will work.